examples/toolkit_showcase/toolkit_showcase.ipynb

```diff
@@ -28,47 +28,12 @@
     "</details>\n",
```

Comment:
<p>"until the end of 2022" --&gt; "2023"</p><p> </p><p>"We don't recommend using this combination in production; instead, wait a few months for Rosemary." --&gt; Maybe consider taking this out? Seems especially redundant with the warning box at the bottom.</p><p> </p><p>Also "FF15SB"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>